Http/Controllers/Api/MediaController.php

```diff
@@ -37,74 +45,100 @@ public function __construct(FileService $fileService, FileRepository $file, Imag
     public function all()
     {
         $files = $this->file->all();
-
+        
         return [
             'count' => $files->count(),
-            'data' => $files,
+            'data' => $files
         ];
     }
 
     /**
      * Store a newly created resource in storage.
-     * @param  UploadMediaRequest $request
+     * 
+     * @param UploadMediaRequest $request            
      * @return Response
      */
     public function store(UploadMediaRequest $request)
     {
         $savedFile = $this->fileService->store($request->file('file'));
-
+        
         if (is_string($savedFile)) {
-            return Response::json(['error' => $savedFile], 409);
+            return Response::json([
+                'error' => $savedFile
+            ], 409);
         }
-
+        
         event(new FileWasUploaded($savedFile));
-
+        
         return Response::json($savedFile->toArray());
     }
 
     /**
      * Link the given entity with a media file
-     * @param Request $request
+     * 
+     * @param Request $request            
      */
     public function linkMedia(Request $request)
     {
         $mediaId = $request->get('mediaId');
         $entityClass = $request->get('entityClass');
         $entityId = $request->get('entityId');
-        $order = $request->get('order');
-
+        
```

Comment:
Sorry, my bad. I made a CTRL+A +V from the wrong source code.
I put again those lines in place. I commited & pushed it.
Do I need to make again the pull request?